api/src/middleware/assessmentsRouter.ts

```diff
@@ -0,0 +1,45 @@
+import { Router } from 'express';
+import { itemEnvelope } from './responseEnvelope';
+
+const assessmentsRouter = Router();
+assessmentsRouter.get('/', (req, res) => {
+  const response = { behaviour: 'list of assessments' };
+  res.status(200).json(itemEnvelope(response));
+});
+
+assessmentsRouter.post('/', (req, res) => {
+    const response = { behaviour: 'create new assessment' };
+    res.status(200).json(itemEnvelope(response));
+  });
+
+assessmentsRouter.get('/:id', (req, res) => {
```

Comment:
I know we used `:id` in the Google Doc, but it'd be better if we were more specific here about what we're expecting. Let's rename this to `:assessmentId` instead.